src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
How much of a problem is that? This SECURITY_IMPERSONATION_LEVEL enumeration does not even have their values assigned to them, they have been assigned automatically by the runtime. Is it something to make the code look better or to add some semantic only? shouldn't we always explicitly assign values to the enums that are used on the unmanaged APIs?